### PR TITLE
Get ready for 3.0

### LINKED ISSUE #179: Version 3.0
### For Version 3

* [x] Remove all adapters except AJ
* [x] Bump required Ruby version to 2.0
* [x] Bump required Rails version to 4.2
* [x] Refactoring
* [x] Cleanup Readme

## For Version 2.10

* [x] Deprecation Warnings https://github.com/jrgifford/delayed_paperclip/pull/178

Anything else?

### PR DESCRIPTION
**Pull request by morgoth**
Fixes #179

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,19 +3,15 @@ Delayed::Paperclip [![Build Status](https://travis-ci.org/jrgifford/delayed_pape
...
-[Resque](https://github.com/resque/resque) or [Sidekiq](https://github.com/mperham/sidekiq).
+attachments in a background task with [ActiveJob](https://github.com/rails/rails/tree/master/activejob)
 
 Why?
 ----
 
 The most common use case for Paperclip is to easily attach image files
 to ActiveRecord models. Most of the time these image files will have
 multiple styles and will need to be resized when they are created. This
-is usually a pretty [slow operation](http://www.jstorimer.com/ruby/2010/01/05/speep-up-your-paperclip-tests.html) and should be handled in a
```
Comment (ScotterC) [inline README.md]: thanks for checking links

File: lib/delayed_paperclip/railtie.rb [hunk 2]
```diff
@@ -1,26 +1,21 @@
-require 'paperclip'
-require 'delayed_paperclip'
+require "paperclip"
+require "delayed_paperclip"
 
 module DelayedPaperclip
-  if defined? Rails::Railtie
```
Comment (ScotterC) [inline lib/delayed_paperclip/railtie.rb]: trying to remember why this was added in the first place - I don't think it'll be a significant problem if it is an issue

**Comment by ScotterC** [comment]
Thanks @morgoth! This brings a lot of simplification. 👍

**Comment by morgoth** [comment]
@ScotterC Feel free to merge and release 3.0

**Pull request merged**